libs/core/datastructures/include/hpx/datastructures/tuple.hpp

```diff
@@ -10,6 +10,27 @@
 
 #pragma once
 
+#if defined(DOXYGEN)
+namespace hpx {
+    // clang-format off
+
+    /// \brief Constructs a tuple that is a concatenation of all tuples in \c tuples.
+    ///        The behavior is undefined if any type in \c std::decay_t<Tuples>...
+    ///        is not a specialization of \c hpx::tuple. However, an implementation
+    ///        may choose to support types (such as \c std::array and \c std::pair)
+    ///        that follow the tuple-like protocol.
+    ///
+    /// \param tuples - zero or more tuples to concatenate
+    ///
+    /// \returns hpx::tuple object composed of all elements of all argument tuples
+    ///          constructed from \c hpx::get<Is>(HPX_FORWARD(UTuple,t) for each
```

Comment:
Is the phrase "constructed by `hpx::get<Is>(HPX_FORWARD(UTuple,t)`" correct?

Reply:
Those are equivalent. `HPX_FORWARD(T, t)` is essentially the same as `std::forward<T>(t)`.